swiperefresh/src/main/java/com/google/accompanist/swiperefresh/SwipeRefresh.kt

```diff
@@ -209,6 +210,8 @@ private class SwipeRefreshNestedScrollConnection(
  * @param indicatorPadding Content padding for the indicator, to inset the indicator in if required.
  * @param indicator the indicator that represents the current state. By default this
  * will use a [SwipeRefreshIndicator].
+ * @param clipIndicatorToPadding Whether to clip the indicator to [indicatorPadding]. If false is
+ * provided the indicator will be clipped to the [content] bounds. Defaults to true.
```

Comment:
will *not* be clipped??

Reply:
Nope, the comment is right. `true` == clip to the padding. `false` the wider bounds.